src/components/BContainer/BContainer.test.js

```diff
@@ -41,4 +41,84 @@ describe('BContainer', () => {
     expect(wrapper.html()).toMatchSnapshot()
     expect(console.error).toHaveBeenCalledWith('Invalid headerLevel property set. Valid input are numbers 1-6.  Reverting to default level 2.')
   })
+  it('shows an arrow if expandable prop is passed', () => {
+    const label = 'hello world'
+    const headerLevel = 3
```

Comment:
Does wrapping this all in `describe` where the constants are set work? 

https://facebook.github.io/jest/docs/en/api.html#describename-fn